pages.es/linux/apk.md

```diff
@@ -0,0 +1,28 @@
+# apk
+
+> Herramienta de gestión de paquetes de Alpine Linux.
+> Más información: <https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management>.
+
+- Actualizar los índices de repositorio desde todos los repositorios remotos:
+
+`apk update`
+
+- Instalar un nuevo paquete:
+
+`apk add {{paquete}}`
+
+- Eliminar un paquete:
+
+`apk del {{paquete}}`
+
+- Reparar paquete o actualizarlo sin modificar dependencias principales:
```

Comment:
This is an old page, we improve our grammar rules in the time. The English page needs to be fixed too
```suggestion
- Repara un paquete o lo actualiza sin modificar dependencias principales:
```